pkg/rpm/salt-minion

```diff
@@ -25,12 +25,11 @@
 # processname: /usr/bin/salt-minion
 
 # Allow these to be overridden for tests
-: ${SALTMINION_BINDIR:=/usr/bin}
-: ${SALTMINION_SYSCONFDIR:=/etc}
-: ${SALTMINION_PYTHON:=python}    # Python must be 2.6 or newer
+: "${SALTMINION_BINDIR:=/usr/bin}"
+: "${SALTMINION_SYSCONFDIR:=/etc}"
```

Comment:
This is a general safety measure to prevent shell globbing and word splitting from occurring there.
If a user will export those variables containing special shell characters such as `*`, `?` ,`[`, `]` or multiple whitespace characters after each other, it will have... undesired effects.

That's maybe a very rear case in this particular piece of code, but I always consider it as good shell coding practice to follow everywhere.